package.json

```diff
@@ -22,6 +22,7 @@
     "now-start": "npm run -s stage:venia",
     "prepare": "node ./scripts/update_repo_environment.js && npx lerna bootstrap --hoist",
     "prettier": "prettier --write '@(packages|scripts)/**/*.@(js|css)' '*.js'",
+    "prettier:validate": "prettier-check '@(packages|scripts)/**/*.@(js|css)' '*.js'",
```

Comment:
I noticed that we have added a prettier-check package to package.json file, right now I added a separate command to run prettier-check, but maybe it makes sense to update our `prettier:check` command. What do you think, guys?

Reply:
Yeah, that's a good improvement--prettier:check is no longer necessary because of prettier --list-different.

Reply:
I like prettier-check package because it analyze the error type and print descriptive message in console. But in general, it is not a big problem, yeah, I can remove `prettier-check` from our package.json file and use our old `prettier:check` command in git `pre-push` hook. Our old `prettier:check` command executes `prettier --list-different` @zetlen, did I understand you correct?

Reply:
rather than call this with `prettier-check` (which didn't work for me locally, can you point to the executable file in the node_modules path so that it can execute the way you are intending here?
`./node_modules/.bin/prettier-check '@(packages|scripts)/**/*.@(js|css)' '*.js'`

if we don't need the `prettier:check` script below this, then we should make an issue in the backlog to refactor its use in the [dangerfile](https://github.com/magento-research/pwa-studio/blob/release/2.0/dangerfile.js#L127-L203), but I would say that is pretty outside the scope of this PR, but should be noted in an issue if it is no longer needed and can be replaced.

Reply:
@sharkySharks, you are right, it is better to point to the executable file in the node_modules folder of the project, thank you. I made changes, please, take a look again.

Reply:
> rather than call this with prettier-check (which didn't work for me locally) can you point to the executable file in the node_modules path so that it can execute

I thought this was done through [package.json's bin field](https://docs.npmjs.com/files/package.json#bin). I'm not super familiar with authoring them myself, but perhaps `prettier-check` hasn't [done it correctly](https://github.com/hexacta/prettier-check/blob/master/package.json#L8)?

Reply:
@Starotitorov - @supernova-at is right, I must not have `npm install`ed so that'd why it wasn't working for me 🤦‍♂️  - feel free to revert back to just `prettier-check`

Reply:
I have updated the pull request.